iib/workers/tasks/build_merge_index_image.py

```diff
@@ -153,31 +155,31 @@ def _add_bundles_missing_in_source(
     )
     _add_label_to_index(
         'com.redhat.index.delivery.distribution_scope',
-        distribution_scope,
+        distribution_scope,  # type: ignore
```

Comment:
It would be nice to put a comment before `_add_label_to_index` indicating the reason for this `type: ignore` like you did for the other ones.

Reply:
Seems that it somehow slipped, this error was removed, with replacing of the `distribution_scope: Optional[str]` for `distribution_scope: str ` in the `_add_bundles_missing_in_source` method.